sample_app/sampleapp_common.c

```diff
@@ -39,6 +39,30 @@ static void print_bytes(uint8_t *bytes, int32_t len)
    printf("\n");
 }
 
+void print_network_details(
```

Comment:
This fetches the details again. Isn't the info already available in the stack state? Could the details change between initializing the stack and calling this function?

Reply:
Now passing already known IP address, MAC address etc instead of reading them again.